qoqo_qiskit_devices/Cargo.toml

```diff
@@ -23,10 +23,10 @@ ndarray = "0.15"
 bincode = "1.3"
 numpy = "0.21"
 
-qoqo = { version = "~1.15", default-features = false }
-roqoqo = "~1.15"
+qoqo = { version = "~1.16", default-features = false }
+roqoqo = "~1.16"
```

Comment:
Can this be updated to 1.19?

Reply:
we are still on pyo3 0.21 here unfortunately, but I have the update for 0.23 stashed